Conclusion.md

```diff
@@ -0,0 +1 @@
+One can only wonder if the Faker Maintainers were recently visited by the Ghost of Faker Past, Ghost of Faker Present, and Ghost of Faker Future. With the recently developed plans for Faker's future, it seems the Maintainers have taken a good long look at the current state of Faker and the previous state of Faker to aid in the trajectory of Faker's future. Ghost of Faker Past, would of shown the Maintainers an early, large, and rigid version of today's Faker. Where as the Ghost of Faker Present would have boasted about the major overhauls, many automations, and the transition from PHP to JS. Completing this eerie dream with the Ghost of Faker Future, the Maintainers would been shown a future where Faker had grown into one large complex object, thus enticing the Maintainers to alter their current trajectory. A new trajectory in which Faker becomes smarter, cleaner, and easier to use. Faker knows where it has been, Faker knows where it is, and Faker knows where it wants to go. 
```

Comment:
In markdown each sentence should be on its own line.

Reply:
Whereas could be made into one word as well.

This could be a "boring" suggestion but the last three sentences "Faker has evolved since its early development, becoming not only easier to develop but to use. With its current direction, Faker is taking a step toward becoming an even better tool for developers to leverage"

Reply:
We could also write this in a different style - Right now its almost like a story, and I wonder how it would sound in a different style (voice)